backend/src/game_objects/hex_objects/wall.rs

```diff
@@ -0,0 +1,26 @@
+use serde::Serialize;
+
+#[derive(Clone, Serialize)]
+pub enum WallKind {
+    Default,
+}
+
+#[derive(Clone, Serialize)]
+pub struct Wall {
+    pub kind: WallKind,
+}
+
```

Comment:
I think, it's too much - to make WallKind now.
More complicated to post from server.
And we will add more new objects before we will add different kinds of one object.